src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -1765,6 +1768,8 @@ void TableDockWidget::readPeakXML(QXmlStreamReader& xml,PeakGroup* parent) {
     p.pos = xml.attributes().value("pos").toString().toInt();
     p.minpos = xml.attributes().value("minpos").toString().toInt();
     p.maxpos = xml.attributes().value("maxpos").toString().toInt();
+    p.splineminpos = xml.attributes().value("splineminpos").toString().toInt();
```

Comment:
@sahil21 This function reads data from mzroll files. Are we handling the case of older versions of .mzroll files which do not have this attribute?

Reply:
@shubhra-agrawal It will work fine for previous versions of mzroll. The spline area will be zero in that case (which is default value)